### PR TITLE
Review app homepage [#LP-69]

### PR DESCRIPTION
**Pull request by rosheen33**
* Custom home page added

connects #LP-69

### CODE REVIEW

File: lms/static/sass/_build-lms-v1.scss [hunk 1]
```diff
@@ -84,4 +84,7 @@
 
 // POC dashboard
 @import 'poc-dashboard';
-@import 'learner-dashboard';
\ No newline at end of file
+@import 'learner-dashboard';
```
Comment (ziafazal) [inline lms/static/sass/_build-lms-v1.scss]: This needs to be in philu's theme

File: lms/static/sass/_homepage.scss [hunk 2]
```diff
@@ -0,0 +1,255 @@
+//
+// OEF Survey Page
+//
+
+/* temp css */
+body {
```
Comment (ziafazal) [inline lms/static/sass/_homepage.scss]: This needs to be in philu's theme.

File: lms/templates/homepage/homepage.html [hunk 3]
```diff
@@ -0,0 +1,234 @@
+## mako
+<%page expression_filter="h"/>
```
Comment (ziafazal) [inline lms/templates/homepage/homepage.html]: This needs to be in philu's theme

**Comment by rosheen33** [comment]
@ziafazal Kindly review again. Design files moved into **philu** theme folder.

File: lms/static/sass/_build-lms-v1.scss [hunk 4]
```diff
@@ -84,4 +84,4 @@
...
```
Comment (ziafazal) [inline lms/static/sass/_build-lms-v1.scss]: @rosheen33 what is the reason for this diff. it should not show up in this changeset.

**Comment by rosheen33** [comment]
I have moved template files and css code to philu theme directory.

**Pull request merged**